src/docs/design/design-audit.mdx

```diff
@@ -0,0 +1,212 @@
+
+<TOC
+    anchors={[
+        { title: 'Introduction', hash: 'ui-design-audits' },
+        { title: 'Prepare for an Audit' },
+        { title: 'Request for a Design Audit' },
+        { title: 'Issue Severity' },
+        { title: 'How to Conduct Design Audits (For Auditors)' },
+        { title: 'Understand Audit Results' },
+        { title: 'Design Audit Examples' },
+    ]}
+/>
+
+# Prepare for an Audit
+
+## Update Packages
+
+Brightlayer UI sometimes makes style updates to components and themes through our NPM packages. Cosmetic issues such as Material component styling can be easily addressed by updating software packages to the latest versions. Therefore, when possible, teams are encouraged to update their Brightlayer UI packages and other major packages to the latest version.
+
+Please refer to our [developer resources page](/resources/developer) to learn about our latest package versions.
+
+Custom component styling is discouraged because the Brightlayer UI’s theme may be suppressed. If the team identifies any styling that should be included as part of the Brightlayer UI component theme, please contact us.
+
+## Read Through Our Website
+
+Please read through our [style guides](/style) and [design pattern guidelines](/patterns). Many examples cited by these guidelines came from design audits we did in the past and are representative of common mistakes we observed across multiple projects. Team members may be able to identify low-hanging fruits during this self-audit and address these problems ahead of the audit.
+
+## Address Comments from Previous Audits
+
+If the team has received audit comments in the past, team members should address these audit comments first before requesting a new one. In case audit comments cannot be addressed due to time / technical constraints, the team should inform their auditors about this.
+
+# Request for a Design Audit
+
+Ideally, a design audit is conducted periodically by someone outside the product team. Team members may [request](/community/contactus) a design audit from the Brightlayer UI team.
+
+## Give a Demo
+
+Team members are expected to walk their auditors through the application to help them understand the big picture. This includes all major workflows of the application.
+
+## Grant Auditors Direct Access
+
+Whenever possible, teams are encouraged to create a safe environment for the auditor to test the project directly. This is to help the auditor explore user interactions and different workflow branches.
+
+## Communicate with User Journey
+
+User experience design is centered around users. Instead of talking about the technical nature of your infrastructure, teams should explain to auditors in terms of user journeys. Who will be using the application? What do they care about the most? How will the application solve this user’s problem?
```

Comment:
```suggestion
User experience design is centered around users. Instead of talking about the technical nature of your infrastructure, teams should explain their product in terms of user journeys. Who will use the application? What do they care about the most? How will the application solve this user’s problem?
```